src/pages/claims/claim-list.tsx

```diff
@@ -0,0 +1,66 @@
+import { NavLink } from 'react-router-dom';
+import Timer, { DATE_FORMAT } from '../../components/timer';
+import { format } from 'date-fns';
+import { images } from '../../utils';
+import { Claim } from '../../chain-data';
+import styles from './claim-list.module.scss';
+
+interface ClaimListProps {
+  claims: Claim[];
+}
+
+export default function ClaimList(props: ClaimListProps) {
+  return (
+    <ul className={styles.claimList}>
+      {props.claims.map((claim) => (
+        <li key={claim.claimId} className={styles.claimItem}>
+          <div className={styles.claimItemMain}>
+            <NavLink className={styles.claimItemTitle} to={`/claims/${claim.claimId}`}>
+              Claim {claim.claimId}
+            </NavLink>
+
+            <div className={styles.claimItemInfo}>
+              {claim.deadline ? (
+                <Timer deadline={claim.deadline} />
+              ) : (
+                <span>{format(claim.timestamp, DATE_FORMAT)}</span>
+              )}
+              <NavLink to={`/policies/${claim.policyId}`}>Policy</NavLink>
+            </div>
+          </div>
+
+          <div className={styles.claimItemStatus}>
+            <ClaimStatus claim={claim} />
```

Comment:
Nit: If the function is only going to return a string or null, you might as well write it as `{getClaimStatus(claim)}` and let the function return string or null.

Reply:
The design is still a bit in flux on the statuses, but I believe we will have some icons or different colours for Rejected / Approved. If that doesn't happen then you are right, this can just be a function.